pkg/cmd/roachtest/main.go

```diff
@@ -287,6 +290,8 @@ runner itself.
 			&count, "count", 1, "the number of times to run each test")
 		cmd.Flags().BoolVarP(
 			&debugEnabled, "debug", "d", debugEnabled, "don't wipe and destroy cluster if test fails")
+		cmd.Flags().BoolVarP(
+			&runSkipped, "run-skipped", "s", runSkipped, "run skipped tests")
```

Comment:
[nit] Omit the "s" shorthand. Leaving it empty avoids adding a short hand altogether.